### PR TITLE
[exporter/tanzuobservability] Turn on metrics exporter.

### PR DESCRIPTION
**Pull request by keep94**
**Description:**
Turn on tanzuobservability metrics exporter

**Testing:** 
Unit tests. Need help with integration testing. As far as I can tell, the only metric types that the OTEL instrumentation API supports are Gauge, Cumulative Sum, Cumulative Histogram (All histograms must share the same explicit boundaries).  I have not found ways to test other metric types such as metric types with delta aggregation temporality and exponential histograms.  Any help with this would be greatly appreciated.

### CODE REVIEW

File: exporter/tanzuobservabilityexporter/metrics_exporter.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+			newSumConsumer(s, settings),
+			newHistogramConsumer(cumulative, delta, s, regularHistogram, settings),
+			newHistogramConsumer(cumulative, delta, s, exponentialHistogram, settings),
+			newSummaryConsumer(s, settings),
+		},
+		s,
+		true), nil
+}
+
+type metricsConsumerCreater func(hostName string, port int, settings component.TelemetrySettings) (
```
Comment (deosu) [inline exporter/tanzuobservabilityexporter/metrics_exporter.go:57]: should this be `metricsConsumerCreator`?

Comment (keep94) [reply exporter/tanzuobservabilityexporter/metrics_exporter.go]: Done. Thanks for catching this.

**Review by deosu — APPROVED** [review]
LGTM

**Comment by keep94** [comment]
Hi. The prometheus compatibility tests and the load tests were failing with this PR.  I don't think these failures are related to my PR. Could they be flaky tests?

**Comment by jpkrohling** [comment]
> Could they be flaky tests?

Yes, they are currently not working.

**Comment by jpkrohling** [comment]
@oppegard @thepeterstone, could one of you please review this PR? Thanks.

**Review by thepeterstone — APPROVED** [review]
LGTM

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**